action.yml

```diff
@@ -1,20 +1,30 @@
 name: 'OpenAPI Spec Diff'
 description: 'A diff tool for OpenAPI Specification 3'
 inputs:
-  format:
-    description: 'Output format'
-    required: false
-    default: 'yaml'
   base:
-    description: 'path of OpenAPI spec in YAML or JSON format'
+    description: 'path of original OpenAPI spec in YAML or JSON format'
     required: true
   revision:
-    description: 'path of OpenAPI spec in YAML or JSON format'
+    description: 'path of revised OpenAPI spec in YAML or JSON format'
     required: true
+  format:
+    description: 'Output format'
+    required: false
+    default: 'yaml'
+  breaking-only:
+    description: 'Display breaking changes only'
+    required: false
+    default: 'false'
```

Comment:
Better to use type: boolean

Reply:
you're right. i started with that, but i do not find that github actions support boolean (only text).
take a look: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions

Reply:
anyway i changed to boolean and it's working ok